ext/bcmath/tests/bcmul_large_numbers.phpt

```diff
@@ -0,0 +1,276 @@
+--TEST--
+bcmul() function with large numbers
+--EXTENSIONS--
+bcmath
+--INI--
+bcmath.scale=0
+--FILE--
+<?php
+require(__DIR__."/run_bcmath_tests_function.inc");
+
+$firstFactors = [
+    "15151324141414.412312232141241",
+    "-132132245132134.1515123765412",
+    "141241241241241248267654747412",
+    "-149143276547656984948124912",
+    "0.1322135476547459213732911312",
+    "-0.123912932193769965476541321",
+];
+$secondFactors = array_merge($firstFactors, [
+    "0",
+    "0.00",
+    "-0",
+    "-0.00",
```

Comment:
As multiplying with 0 is already done in another test those should be removed IMHO